chainer/links/loss/crf1d.py

```diff
@@ -12,17 +13,24 @@ class CRF1d(link.Link):
 
     Args:
         n_label (int): Number of labels.
+        initialW (:ref:`initializer <initializer>`): Initializer to
+            initialize the weight. When it is not specify,
+            the weight is initialized with zero.
```

Comment:
How about using "transition cost matrix" instead of "weight", as it better describes the target parameter?

Reply:
Thanks for the careful suggestion, I updated :bow: